sapl/painel/views.py

```diff
@@ -368,17 +368,21 @@ def get_presentes(pk, response, materia):
     sessao = SessaoPlenaria.objects.get(id=pk)
     num_presentes = len(presentes)
     data_sessao = sessao.data_inicio
-    oradores = OradorExpediente.objects.filter(
-        sessao_plenaria_id=pk).order_by('numero_ordem')
-
+    
+    if type(materia) is ExpedienteMateria:
```

Comment:
A gente usa `isinstance` ao invés de type no código. Tem a vantagem de retornar True se objeto herdar de uma classe 'X' ou for uma subclasse de 'X'.

```suggestion
    if isinstance(materia, ExpedienteMateria):
```

Reply:
Ajustado